diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -193,7 +196,8 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule(
         list.forEachIndexed { index, astNode ->
             val dotNode = astNode.getFirstChildWithType(DOT) ?: astNode.getFirstChildWithType(SAFE_ACCESS)
             val nodeBeforeDot = dotNode?.treePrev
-            if (index > 0) {
+            val firstElem = astNode.firstChildNode
+            if (index > 0 && (firstElem.textContains('(') || firstElem.textContains('{'))) {
```

Comment:
Expression `firstElem.textContains('(') || firstElem.textContains('{')` occurs 3 times in this class already. It's time to extract it into a function

Reply:
done